### PR TITLE
✨ AddRider Page plus function

### PR DESCRIPTION
**Pull request by bwnstck**
# Description

as a part of #130 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by arassaria — APPROVED** [review]
LGTM 🚀 🐓

**Review by arassaria — APPROVED** [review]
LGTM 🚀 🚴🏻

File: client/src/pages/AddRider.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components/macro";
+import { useHistory, useLocation, useParams } from "react-router-dom";
```
Comment (arassaria) [inline client/src/pages/AddRider.js]: useLocation is unused

File: client/src/pages/AddRider.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+import {
+  addData,
+  addTour,
```
Comment (arassaria) [inline client/src/pages/AddRider.js]: unused

File: client/src/pages/AddRider.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  getDataByID,
+  getSortedData,
```
Comment (arassaria) [inline client/src/pages/AddRider.js]: unused

File: client/src/pages/AddRider.js [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  updateData,
+} from "../utils/api";
+import DefaultAvatar from "../assets/defaultAvatar.svg";
+
+import Badge from "../components/Badge";
+import Card from "../components/Card";
+import HeaderMain from "../components/HeaderMain";
```
Comment (arassaria) [inline client/src/pages/AddRider.js]: unused

File: client/src/pages/AddRider.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+import Input from "../components/Input";
+import Button from "../components/Button";
+import InfoInput from "../components/InfoInput";
```
Comment (arassaria) [inline client/src/pages/AddRider.js]: unused

**Review by arassaria — CHANGES_REQUESTED** [review]
Check for unused vars. Read build log if i missed any

**Review by bwnstck — COMMENTED** [review]
Thx

**Pull request merged**